GA-worksheet.md

```diff
@@ -26,21 +26,21 @@
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
-
+The link tag is a link to something or a heading. It also makes a specific style.
 2. What is normalize.css?
-
+Normalize.css renders all elements more constantly and in line.
```

Comment:
:+1: It makes sure that all your CSS is rendered consistently throughout all browsers